data/transactions/logic/opcodes_test.go

```diff
@@ -81,7 +81,8 @@ func TestOpcodesByVersionReordered(t *testing.T) {
 }
 
 func TestOpcodesByVersion(t *testing.T) {
-	partitiontest.PartitionTest(t)
+	// partitiontest.PartitionTest(t)
+	// has partitioning in the TestOpcodesByVersionReordered()
```

Comment:
Cool, you caught a real double-sharding bug!

Reply:
thank you for your help with this! couldn't have done it without you!